pennylane/tape/qscript.py

```diff
@@ -291,7 +293,7 @@ def observables(self) -> List[Union[MeasurementProcess, Observable]]:
         return obs
 
     @property
-    def measurements(self) -> List[MeasurementProcess]:
+    def measurements(self) -> List[Union[SampleMeasurement, StateMeasurement]]:
```

Comment:
Can this also be `MeasurementTransform`? I think keeping this as `MeasurementProcess` is more readable.

Reply:
Yep you're right. I added the `Union` because the `MeasurementProcess` class doesn't know about the `process_samples` and `process_state` methods. 

Changed to `MeasurementProcess` to make it more readable 👍